core/ecschema-editing/src/test/Merging/ClassMergerOrder.test.ts

```diff
@@ -0,0 +1,367 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import {  EntityClass, Schema, SchemaContext, StructClass } from "@itwin/ecschema-metadata";
+import { SchemaMerger } from "../../ecschema-editing";
+import { expect } from "chai";
+
+describe("Struct class merger tests", () => {
```

Comment:
Should all these tests be within the 'Struct class merger tests' group?  You also have a lot of sub describes that just have one test.  I think the type of thing the test applies to should be clear even if it was a flat list of tests under the top level describe